sql/SQLCode.sql

```diff
@@ -0,0 +1,26 @@
+CREATE User(
+user_ID VARCHAR(5),
+u_name VARCHAR(30) NOT NULL,
+u_pwd VARCHAR(30) NOT NULL,
+is_admin BIT DEFAULT 0,
+PRIMARY KEY (user_ID)
```

Comment:
```suggestion

```

Reply:
Same for the other tables, this works but I think its overly verbose when you can also use the short syntax.
As shown above